src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/PascalCasedSerializableImageElement.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Runtime.Serialization;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Completion
+{
+    [DataContract]
+    internal class PascalCasedSerializableImageElement
```

Comment:
I'm sure it was intentional, but why switch this to a class?

Reply:
So I can do a reference lookup for the icon cache 😄